### DISCUSSION TITLE
Directory servers Any Genre1/2/3 issues?

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#2900] [Technical Support & Help] [answered]
Are the directory servers for Any Genre1/2/3 having issues at the moment?
explorer.jamulus.io shows no data from each genre.

**Comment by rdica** [comment, marked as answer]
It looks like they've returned, but they were out of reach for about 15-20 minutes.

**Reply by ann0see** [reply]
Probably maintenance. I have a server with the same provider and got a mail saying some of their servers had issues:

> aufgrund eines Problems mit unserem Kernel Live-Patching müssen wir die Hostsysteme xxx kurzfristig rebooten.

> Bedingt durch den Reboot kommt es zu einer kurzen Downtime Ihres VPS von ca. 5 - 10 Minuten. Wir beginnen mit den Reboots gegen ca. 22:00 Uhr und werden die Server nacheinander rebooten.
